Oop/OOP/Mammals.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OOP
+{
+
+    public class Mammals : Animals
+    {
```

Comment:
la fel si aici...nu are nimic extra, nimic in plus, nu e abstracta...e copy paste anmals=>nu ai nevoie de ea